snippets/ruby-rspec/desc.snippet

```diff
@@ -1,5 +1,3 @@
-require 'spec_helper'
-
-describe ${1:`Snippet_RubyClassNameFromFilename()`} do
+describe '${1}' do
 	${2}
 end
```

Comment:
There is a [des](https://github.com/activebridge/rails-snippets/blob/master/snippets/ruby-rspec/des.snippet) snippet for that